src/constants/constants.js

```diff
@@ -9,12 +9,20 @@ export const API_AUTH_PASS = process.env.API_AUTH_PASS;
 export const API_BASE_URL = '/api'; //proxed to 'https://pfa.foreca.com' via webpack
 export const API_AUTH_URL = '/authorize/token';
 export const API_SEARCH_URL = '/api/v1/location/search/';
+export const API_GET_LOCATION_INFO_URL = '/api/v1/location/';
 export const API_CURRENT_WEATHER_URL = '/api/v1/current/';
 export const API_FORECAST_URL = '/api/v1/forecast/';
-export const API_FORECAST_3_HOURLY = '3hourly/';
+export const API_FORECAST_DETAILED = '3hourly/';
```

Comment:
api forecast daily ...what? puppies? 

would it be beneficial if you would have 
```
getHourlyForecastURL= hoursNumber => `${hoursNumber}hourly/`;
```
?

Reply:
In this case no. We'll loose at least in network traffic.
API has two fixed endpoints '3hourly/' and 'hourly/'. There are no `${OTHER_NUMBER}hourly/` endpoints on server.
In theory I can implement getAnyHourly function on base of 'hourly/' endpoint - get all hours data and filter only needed hours. But in this case we'll seriously loose in network traffic because the largest part of received data will be filtered.
So my point is to use the current implementation with '3hourly/' endpoint because it gives exactly needed amount of data.

Reply:
ok that's fine. I just thought that maybe they have more flexible endpoints for that purpose.